### DISCUSSION TITLE
Can anyone test if this is a bug or not?

### DISCUSSION THREAD

**Discussion by Trung0246** [ppy/osu#27413] [Q&A]
Requirements: 2 monitors (or more).
OS: Windows 11

Reproduce step:
1) Check which is the main monitor on windows 11.
2) Once checked, open osu!lazer (by doubleclick on the exe or shortcut) on secondary monitor (NOT main monitor).
3) Game should be open on the main monitor.

However once you started playing a song, the outdated warning popup keep occur and therefore no score will be submitted. The only way to fix this is to close the game then actually open the game in the main monitor.

Version: 2024.221.0

This is a very weird problem I encountered when testing my new monitor just bought last week.

**Comment by bdach** [comment]
No I cannot reproduce this, and please attach logs, because this sounds completely bonkers.
